examples/satellites/systems.mojo

```diff
@@ -0,0 +1,63 @@
+from random import random
+from larecs import World
+from components import Position, Velocity
+from parameters import Parameters, GRAVITATIONAL_CONSTANT
+from python import PythonObject, Python
+
+
+fn move(mut world: World, parameters: Parameters) raises:
+    for properties in world.get_entities[Position, Velocity]():
+        position = properties.get_ptr[Position]()
+        velocity = properties.get_ptr[Velocity]()
+
+        position[].x += velocity[].x * parameters.dt
+        position[].y += velocity[].y * parameters.dt
+
+
+fn accellerate(mut world: World, parameters: Parameters) raises:
```

Comment:
This pattern will make it hard to come up with a `Systems` interface that is generally useful (in case you want to add systems to the library at all).

It is a good candidate for a [resource](https://mlange-42.github.io/arche/guide/resources/).